spring-cloud-dataflow-docs/src/main/asciidoc/applications.adoc

```diff
@@ -11,133 +11,136 @@ A selection of pre-built link:http://cloud.spring.io/spring-cloud-stream-app-sta
 |======================
 |Source |Processor |Sink |Task
 
-|link:{scs-app-starters-docs-htmlsingle}/#spring-cloud-stream-modules-sftp-source[sftp]
-|link:{scs-app-starters-docs-htmlsingle}/#spring-cloud-stream-modules-tcp-client-processor[tcp-client]
-|link:{scs-app-starters-docs-htmlsingle}/#spring-cloud-stream-modules-mqtt-sink[mqtt]
-|link:https://docs.spring.io/spring-cloud-task-app-starters/docs/Clark.RELEASE/reference/htmlsingle/#_spark_yarn_task[spark-yarn]
+|link:{scs-app-starters-docs-htmlsingle}/#spring-cloud-stream-modules-sftp-source[sftp] ~(1.3.x,2.0.x,2.1.x)~
```

Comment:
It isn't clear to me what adding (1.3.x,2.0.x,2.1.x) is adding (it isn't even clear it is for a version of spring cloud stream).


I think it would be easier to have the information bullet below this table say that 'each application listed below is 
available in the Celsius, Darwin, and Einstein Spring Cloud App Starters Release trains unless otherwise noted'  We put in parenthesis (deprecated) for a few apps.  This make the table less cluttered.  A small bullet list of 

- Celsius - Based on Spring Boot 1.5.x + Spring Cloud Stream 1.3.x
- Darwin - Based on Spring Boot 2.0.x + Spring Cloud Stream 2.0.x
- Einstein - Spring Boot 2.1.x + Spring Cloud Stream 2.1.x

Reply:
Also, @markpollack, a few of the apps don't have all the 3 versions supported, so it does require some form of a hinting as to what is supported. That's one of the reason I chose the parenthesis path.

If you think we don't want to advertise 1.3.x Apps anymore, we could do what you're suggesting. Thoughts?

Reply:
And, btw, the rendering of parenthesis is a subscript, so it looks like this:

![image](https://user-images.githubusercontent.com/1562654/52129567-ae66e800-25ec-11e9-8c69-7d8fde1069da.png)